composer.json

```diff
@@ -105,7 +105,8 @@
         "php-http/httplug":               "^1.0",
         "php-http/client-common":         "^1.3",
         "php-http/message-factory":       "^1.0",
-        "php-http/discovery":             "^1.0"
+        "php-http/discovery":             "^1.0",
+        "ext-json": "^1.5"
```

Comment:
spaces required

Reply:
I'm not going to manually edit `composer.json`. It should be a generated file.